### DISCUSSION TITLE
Plotter Class

### DISCUSSION THREAD

**Discussion by GokhanPay** [dstl/Stone-Soup#597] [Q&A]
Hi,

I am trying to run the sample codes from the page "An introduction to Stone Soup: using the Kalman filter (https://stonesoup.readthedocs.io/en/v0.1b7/auto_tutorials/01_KalmanFilterTutorial.html#an-introduction-to-stone-soup-using-the-kalman-filter)" as a .py file using Anaconda & VS code. The code works properly (I can print and see the results) but the plots (using methods of Plotter class) does not show. I thing the class is missing .show() function inside. I also tried to run the codes from VS code python terminal but the problem persisted. Do you have any idea to resolve it ?

**Comment by sdhiscocks** [comment]
I think this is an issue when using this outside Jupyter Notebooks. You should be able to use `plt.show()` or `plotter.fig.show()`
